src/aks-preview/azext_aks_preview/_resourcegroup.py

```diff
@@ -0,0 +1,13 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from ._client_factory import cf_resource_groups
+
+
+def get_rg_location(ctx, resource_group_name, subscription_id=None):
+    groups = cf_resource_groups(ctx, subscription_id=subscription_id)
+    # Just do the get, we don't need the result, it will error out if the group doesn't exist.
+    rg = groups.get(resource_group_name)

```

Comment:
Hi Andy, a lot of this code is not mine. It was simply moved around for organization purposes and to avoid circular dependencies. I ensured everything still works, but did not change any code functionality except for functions that were directly related to addons (and there were only 2 of them). I'm not sure it would be safe for me to change the functionality of these other functions without knowing there purpose.